src/jit/compiler.h

```diff
@@ -252,8 +252,9 @@ class LclVarDsc
     unsigned char lvStackByref : 1;  // This is a compiler temporary of TYP_BYREF that is known to point into our local
                                      // stack frame.
 
-    unsigned char lvArgWrite : 1; // variable is a parameter and STARG was used on it
-    unsigned char lvIsTemp : 1;   // Short-lifetime compiler temp
+    unsigned char lvArgWrite : 1;         // there is at least one STLOC or STARG on this local
+    unsigned char lvMultipleArgWrite : 1; // there is more than one STLOC on this local
```

Comment:
> I'd use definition but then the overlap with lvISingleDef would be even more confusing. How about HasSingleWrite and HasMultipleWrites?

I thought about definition and the same thing occurred to me.  I picked "store" because it's the verb used in the IL opcodes, and to my mind they key point is that these properties track which IL opcodes get used and how often, so that you can take advantage of them when importing that single IL operation... I don't think that "write" conveys that (though I'll admit that "store" doesn't do a great job of conveying it).  Maybe something along the lines of `lvSingleILDef` or `lvSingleILStoreOp`?

> I could resolve this by setting lvArgWrite when the arg temp is allocated and update the logic for args and then the fields would have same meaning. Does that seem preferable?

maybe?  On the one hand, I feel ok with what you have (modulo naming).  On the other hand, I had to read over the change a few times to figure out that the asymmetry was intentional and because of the implicit arg def at the callsite... I guess whatever fits with the names is preferable to me, so if you keep them with names matching the opcodes then like you have it seems preferable, but if you switch to something more generic like `Def`/`Write` then yeah, reworking it to count the implicit arg defs at callsites as defs/writes would make more sense.